Source/Core/Core/HW/WiimoteEmu/Extension/DrawsomeTablet.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/Swap.h"
+#include "Core/HW/WiimoteEmu/Extension/Extension.h"
+
+namespace ControllerEmu
+{
+class AnalogStick;
+class Triggers;
+class ControlGroup;
+}  // namespace ControllerEmu
+
+namespace WiimoteEmu
+{
+enum class DrawsomeTabletGroup
+{
+  Stylus,
+  Touch,
+};
+
+class DrawsomeTablet : public Extension3rdParty
+{
+public:
+  DrawsomeTablet();
+
+  void Update() override;
+  bool IsButtonPressed() const override;
+  void Reset() override;
+
+  ControllerEmu::ControlGroup* GetGroup(DrawsomeTabletGroup group);
+
+  struct DataFormat
+  {
+    // Pen X/Y is little endian.
+    u8 stylus_x1;
+    u8 stylus_x2;
+
+    u8 stylus_y1;
+    u8 stylus_y2;
+
+    u8 pressure1;
+
+    u8 pressure2 : 3;
```

Comment:
IIRC, new code should prefer BitField over C bitfields.

Reply:
Fixed.